NAMESPACE

```diff
@@ -31,7 +31,7 @@ export(style_text)
 export(tidyverse_math_token_spacing)
 export(tidyverse_reindention)
 export(tidyverse_style)
-importFrom(magrittr,"%>%")
+importFrom(magrittr,)
```

Comment:
This was committed by precommit, but I am pretty sure it's a bug in the new version of _roxygen2_.

Reply:
Let’s revert that then, when you file an issue or find one, We can link it to you.

Reply:
Never mind. I adapted our code to the new API, so no need to file any issue upstream.

Reply:
For future reference, this was the original issue here: https://github.com/r-lib/roxygen2/issues/1570